projectpage/table

```diff
@@ -0,0 +1,18 @@
+<?php
+$connect = new mysqli("localhost", "root", "", "search");
+if($connect != null){
+ echo "connected";
+}else {
+ echo "failed to connect ";
+}
+
+$sqlQuery = "SELECT * FROM takes";
+$qry = $connect->query($sqlQuery);
+
+echo "<table><tr><th></th><th>search_name</th><th>date</th><th>category</th><th>;
+while($row = $qry->fetch_assoc()){
+echo "<tr><td>".$row[""]."</td><td>".$row["search_name"].$row["date"]."</td><td>".$row["category"]."</td>;
+}
+
+echo "</table>";
+?>
```

Comment:
are you using php?

Reply:
We aren't using php, but we were going to. We will delete it.